### PR TITLE
Build with Android 15 preview SDK

### PR DESCRIPTION
**Pull request by FooIbar**

### CODE REVIEW

**Comment by VDostoyevskiy** [comment]
Why do we do that in such a hurry?
I have neither a Android 15 phone nor archlinuxcn package for it

**Comment by FooIbar** [comment]
> Why do we do that in such a hurry? I have neither a Android 15 phone nor archlinuxcn package for it

We are not targeting Android 15 (can't either), just building with it .
Aren't SDKs downloaded through sdkmanager? Why would archlinuxcn packages be required.

**Comment by VDostoyevskiy** [comment]
Seems artifacts are invalid?
PackageManager failed to install it

**Comment by FooIbar** [comment]
> Seems artifacts are invalid? PackageManager failed to install it

Might be a combined effect of the R8 update, since I did successfully installed debug build.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -56,6 +56,9 @@ jobs:
           key: ${{ runner.os }}-cmake-${{ hashFiles('gradle/libs.versions.toml', 'app/build.gradle.kts', 'app/src/main/cpp/CMakeLists.txt') }}
           restore-keys: ${{ runner.os }}-cmake-
 
+      - name: Install Build Tools
+        run: echo "y" | ${ANDROID_SDK_ROOT}/cmdline-tools/latest/bin/sdkmanager --install "build-tools;35.0.0-rc1"
```
Comment (VDostoyevskiy) [inline .github/workflows/ci.yml:60]: Why do we need to install this manually?

**Review by VDostoyevskiy — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (FooIbar) [reply .github/workflows/ci.yml:60]: Apparently preview SDKs have a different license than those pre-installed ones https://github.com/FooIbar/EhViewer/actions/runs/7940239629/job/21681254875

**Pull request merged**